test/k8s-integration/main.go

```diff
@@ -487,7 +533,7 @@ func runTestsWithConfig(testDir, testFocus, testSkip, testConfigArg string, clou
 	}
 	ginkgoArgs := fmt.Sprintf("--ginkgo.focus=%s --ginkgo.skip=%s", testFocus, testSkip)
 	if *platform == "windows" {
-		ginkgoArgs = ginkgoArgs + fmt.Sprintf(" --node-os-distro=%s", *platform)
+		ginkgoArgs = ginkgoArgs + fmt.Sprintf(" --node-os-distro=%s --allowed-not-ready-nodes=3", *platform)
```

Comment:
Can you add a comment explaining why there are not ready nodes?

Reply:
added

Reply:
added